File: C Programming/Competitive Programming/Coding Challengs 17/program5.c
/*
5. Accept number of rows and number of columns from user and display
below pattern.
Input : iRow = 3 iCol = 4
Output : 1 1 1 1
         2 2 2 2
         3 3 3 3
*/

#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0;
    int j = 0;

    printf("Output is :\n");

    // Outer Loop -> Rows
    for(i = 1; i <= iRow; i++)
    {
        if (i==1)
        {
            printf("Output is : ");
        }
        else
        {
            printf("            ");//12 spaces
        }
        // Inner Loop -> Columns
        for(j = 1; j <= iCol; j++)
        {
            printf("%d ", i);
        }

        printf("\n");
    }
}

int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    printf("Enter Number Of Rows : ");
    scanf("%d", &iValue1);

    printf("Enter Number Of Columns : ");
    scanf("%d", &iValue2);

    Pattern(iValue1, iValue2);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 6/program5.java
import java.util.Scanner;

class program5 
{
    public static void Pattern(int iRow,int iCol) 
    {
        int i=1;
        int j=0;

        do 
        {
            j=1;
            do 
            {
                if (i == 1 || i == iRow) 
                    { 
                        System.out.print(j + " "); 
                    } 
                else if (j == 1) 
                    { 
                        System.out.print(1 + " "); 
                    } 
                else if (j == iCol) 
                    { 
                        System.out.print(iCol + " "); 
                    } 
                else 
                    { 
                        System.out.print("* "); 
                    }
                    j++;
            } 
            while (j <= iCol);
            System.out.println("");
            i++;
        } 
        while ( i <= iRow);
    }
    public static void main(String[] args) 
    {
    Scanner sobj=new Scanner(System.in);
    int iValue1=0,iValue2=0;

    System.out.print("Enter The Number of Row: ");
    iValue1=sobj.nextInt();

    System.out.print("Enter The Number of Column: ");
    iValue2=sobj.nextInt();

    Pattern(iValue1,iValue2);

    sobj.close();
    }
}
